### PR TITLE
Newsletters: add 199 (2022-05-11)

### PR DESCRIPTION
**Pull request by harding**
- [x] Bitcoin PR Review Club meeting summary @glozow 
- [x] Rust Bitcoin 716 - amount:display formatting @adamjonas 
- [x] Releases and RCs @harding 
- [x] Topic links @harding 

There's not really any news yet.  If I see anything easy in the next couple days, I'll add that, otherwise it'll be an easy newsletter.

### CODE REVIEW

**Comment by Xekyo** [comment]
Indeed, very easy review so far. ;)
SGTM

**Comment by adamjonas** [comment]
No comments from me either. Flawless @harding

File: _posts/en/newsletters/2022-05-11-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+relevant.
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review Club][]
+meeting, highlighting some of the important questions and answers.  Click on a
+question below to see a summary of the answer from the meeting.*
+
+[Improve Indices on pruned nodes via prune blockers][review club pr] is a PR by Fabian Jahr to
+introduce a new method for deciding when it is safe to prune a node from block storage. This new
```
Comment (nechteme) [inline _posts/en/newsletters/2022-05-11-newsletter.md:28]:
```suggestion
introduce a new method for deciding when it is safe to prune a block from block storage. This new
```

File: _posts/en/newsletters/2022-05-11-newsletter.md [hunk 2]
```diff
@@ -24,14 +24,39 @@ relevant.
...
+  a0link="https://bitcoincore.reviews/21726#l-28"
 
+  q1="What is a circular dependency? Why do we want to avoid them when possible?"
+  a1="A circular dependency between two code modules exists when both cannot be used without the
+other. While circular dependencies are not a security issue, they signify poor code organization
+and encumber development by making it more difficult to build, use, and test specific modules or
+functionality in isolation."
+  a1link="https://bitcoincore.reviews/21726#l-44"
+
+  q2="How do the prune blockers introduced in [this commit][review club commit] work?"
```
Comment (Xekyo) [inline _posts/en/newsletters/2022-05-11-newsletter.md:48]: @glozow: Are "prune blockers" and "prune locks" the same things? If so, let's use only one term.

**Comment by harding** [comment]
Pushed edits, thanks everyone!  Left https://github.com/bitcoinops/bitcoinops.github.io/pull/756#discussion_r869483266 for @glozow to decide whether to edit or not.

No releases/RCs this week, so I removed that section.  There were also no topic mentions, so I didn't add any topic links.

I've already started writing next week's newsletter and it looks lie it might be a long one, so hopefully y'all enjoyed the short one this week.  :-)

**Comment by bitschmidty** [comment]
Going to leave https://github.com/bitcoinops/bitcoinops.github.io/pull/756#discussion_r869483266 for now and @glozow can address in a follow up PR if necessary.

File: _posts/en/newsletters/2022-05-11-newsletter.md (hunk 2 above)
Comment (jonatack) [reply _posts/en/newsletters/2022-05-11-newsletter.md:51]: > @glozow: Are "prune blockers" and "prune locks" the same things? If so, let's use only one term.

IIRC the "prune blockers" naming was replaced in a later push by prune locks.

**Comment by bitschmidty** [comment]
Pushed a small spelling correction and squashed.

A newsletter for 🐜 s? Thanks @harding @glozow @Xekyo @adamjonas @nechteme !

**Pull request merged**